README.md

```diff
@@ -9,16 +9,25 @@
 
 # The Open Source Observability Distribution
 
-Features that are not guarded behind a paywall.
```

Comment:
I think this is not the kind of language we want to have in our main README. Would love for us to discuss async maybe. I really propose to just remove it for now :).

Reply:
That's fine.

Reply:
Same. The idea is to be clear that this project will only be open:-)

Reply:
OK, we'll find better wording!